### PR TITLE
testing the modification

### PR DESCRIPTION
**Pull request by EGAREDAME**

### CODE REVIEW

File: work3.py [hunk 1]
```diff
@@ -6,4 +6,5 @@ def get_middle_three_chars(str1):
     print("Middle three chars are:", res)
 
 get_middle_three_chars("JhonDiePeta")
-get_middle_three_chars("JaSonAy")
\ No newline at end of file
+get_middle_three_chars("JaSonAy")
+#comment
```
Comment (nkawuv) [inline work3.py:10]: was this change necessary - i dont see any significant change just a comment added

Comment (EGAREDAME) [reply work3.py:10]: LOL

**Review by nkawuv — APPROVED** [review]
Looks good to me
